language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeBetaIT.java

```diff
@@ -35,7 +35,7 @@
 public class AnalyzeBetaIT {
 
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
-  private static final String BUCKET = PROJECT_ID;
+  private static final String BUCKET = "cloud-samples-data";
```

Comment:
Do you need this if you are hard-coding the link to the bucket?

Reply:
Nope